conda/build_cpu.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+conda build tvm-libs
```

Comment:
For robustness it is good practice to include at least:
set -e
set -u
at the top of every .sh script.  This will ensure that if any command in the script fails, the error is propagated rather than elided away, it will also ensure that scripts using undefined variables fail.

Reply:
Ok, I'll do that.